src/hooks/data/useData/index.tsx

```diff
@@ -244,6 +244,11 @@ async function fetchData(
       rawResponse: response,
     });
   } catch (error) {
+    if (error.response.status === 401) {
```

Comment:
Is the user then going to be presented with the login automatically again?

In that case, a message would be useful such as "Session expired, please login again".

Reply:
thanks @mberacochea . 

I've added an alert to indicate this. 

let me know your thoughts:

![Uploading Screenshot 2024-10-04 at 15.58.46.png…]()

Reply:
<img width="1246" alt="Screenshot 2024-10-04 at 15 58 46" src="https://github.com/user-attachments/assets/ea2b8186-8d79-4c95-a95b-a0cc5ab1b6f7">

Hi @mberacochea let me know if you can see the screenshot now

Reply:
yeah, I can see the message now. IMO a VF [info banner](https://stable.visual-framework.dev/components/vf-banner/index.html?q=banner#vf-banner--info) would be better, they are less obstructive.

Reply:
thanks @mberacochea I've added the banner as suggested. 
made a couple more changes to make the login form use a primary button and a loading spinner